src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Yep, the names in the menus are still crappy too and should be changed. If you have a traditional FreeDOS, you should first choose "2. Install user provided DOS." then choose "1. Install provided DOS.". Then the menu with the dosses from the script shows (this works ok). Then you choose either 1 or 2 depending on whether you want FreeDOS or MS-DOS. This will trigger the scripts to download, extract and install. None of this output shows up, even though I expect the code in install.c to print it (line 259 and line 273). The script, while invisible, do execute correctly though.